app/views/rubygems/show.html.erb

```diff
@@ -1,7 +1,12 @@
 <% @title = @rubygem.name %>
 <% @subtitle = @latest_version.try(:slug) %>
 
-<%= content_for :head, auto_discovery_link_tag(:atom, rubygem_versions_path(@rubygem, format: "atom"), {title: "#{@rubygem.name} Version Feed"}) %>
+<% content_for :head do %>
+  <%= auto_discovery_link_tag(:atom, rubygem_versions_path(@rubygem, format: "atom"), {title: "#{@rubygem.name} Version Feed"}) %>
+  <% unless @latest_version.indexed %>
```

Comment:
Look at the failing tests, but I think the ivar `@latest_version`  is not always set. So if you dont specify a version, that var will be nil, in that case we should not add the metatag.